flink-runtime/src/test/java/org/apache/flink/runtime/jobmaster/slotpool/SlotPoolImplTest.java

```diff
@@ -893,14 +866,6 @@ protected boolean matchesSafely(AllocatedSlotInfo item, Description mismatchDesc
 		};
 	}
 
-	private CompletableFuture<LogicalSlot> allocateSlot(Scheduler scheduler, SlotRequestId slotRequestId) {
-		return scheduler.allocateSlot(
-			slotRequestId,
-			new DummyScheduledUnit(),
-			SlotProfile.noRequirements(),
-			timeout);
-	}
-
 	private SlotPoolImpl createAndSetUpSlotPool() throws Exception {
```

Comment:
Let's move `createSlotPoolImpl()` to be below this method because it is first (and only) used here.
Also let's remove the `@Nonnull` tag of it because it is against current code style.